frontend/__mocks__/mutationObserver.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+
+global.MutationObserver = global.MutationObserver || class {
+  constructor(callback) {}
+  disconnect() {}
+  observe(element, initObject) {}
+};
+
```

Comment:
Nit: two trailing newlines, one is enough :smiley:

Reply:
done